### PR TITLE
Package coq-tree-calculus.1.0.0

### PR DESCRIPTION
**Pull request by barry-jay-personal**

### CODE REVIEW

File: released/packages/coq-tree-calculus/coq-tree-calculus.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+homepage: "https://github.com/barry-jay-personal/tree-calculus"
+dev-repo: "git+https://github.com/barry-jay-personal/tree-calculus.git"
+bug-reports: "https://github.com/barry-jay-personal/tree-calculus/issues"
+doc: "https://barry-jay-personal.github.io/tree-calculus"
+maintainer: "barry.jay8@gmail.com"
+authors: [
+  "Barry Jay"
+]
+license: "MIT" # Make sure this is reflected by a LICENSE file in your sources
```
Comment (clarus) [inline released/packages/coq-tree-calculus/coq-tree-calculus.1.0.0/opam:24]: Forgotten comment

File: released/packages/coq-tree-calculus/coq-tree-calculus.1.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  src: "https://github.com/barry-jay-personal/tree-calculus/archive/1.0.0.tar.gz"
+  checksum: "sha256=eb5b8bbc81e27c783334ff9d17d910495aa14b4c46b9a3465a85712b163ac33f"
+}
+
+tags: [
+  "keyword:tree calculus"
+  "keyword:reflective programs"
+  "categoryCS/Lambda Calculi"
+  "date:2020-09-20"
+  "logpath:tree-calculus"
```
Comment (clarus) [inline released/packages/coq-tree-calculus/coq-tree-calculus.1.0.0/opam:47]: Isn't the `logpath` (corresponding to where the files are required from) `Reflective` according to `_CoqProject`?

File: released/packages/coq-tree-calculus/coq-tree-calculus.1.0.0/opam (hunk 1 above)
Comment (barry-jay-personal) [reply released/packages/coq-tree-calculus/coq-tree-calculus.1.0.0/opam:24]: Is there a problem? I have a LICENSE file in the repository and a copy of the license in each file.

File: released/packages/coq-tree-calculus/coq-tree-calculus.1.0.0/opam (hunk 2 above)
Comment (barry-jay-personal) [reply released/packages/coq-tree-calculus/coq-tree-calculus.1.0.0/opam]: Thanks! I have made the change to the file opam and pushed it, using `git push origin coq-tree-calculus.1.0.0` Is this enough or do I need to make a new pull request too? (I'm new to collaborating through GIT).

File: released/packages/coq-tree-calculus/coq-tree-calculus.1.0.0/opam [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+url {
+  src: "https://github.com/barry-jay-personal/tree-calculus/archive/1.0.0.tar.gz"
+  checksum: "sha256=eb5b8bbc81e27c783334ff9d17d910495aa14b4c46b9a3465a85712b163ac33f"
+}
+
+tags: [
+  "keyword:tree calculus"
+  "keyword:reflective programs"
+  "categoryCS/Lambda Calculi"
```
Comment (palmskog) [inline released/packages/coq-tree-calculus/coq-tree-calculus.1.0.0/opam:45]:
```suggestion
  "category:Computer Science/Lambda Calculi"
```

Comment (barry-jay-personal) [reply released/packages/coq-tree-calculus/coq-tree-calculus.1.0.0/opam]: Thanks! I have made the change to the file opam and pushed it, using git push origin coq-tree-calculus.1.0.0 Is this enough or do I need to make a new pull request too? (I'm new to collaborating through GIT).

Comment (palmskog) [reply released/packages/coq-tree-calculus/coq-tree-calculus.1.0.0/opam]: @barry-jay-personal all you need to do is push changes to the branch, as you already did. The only situation I can see for making a new PR is when there is no chance the current PR will be accepted by maintainers (not very common for the project).

**Comment by barry-jay-personal** [comment]
Great, thx!

File: released/packages/coq-tree-calculus/coq-tree-calculus.1.0.0/opam (hunk 1 above)
Comment (barry-jay-personal) [reply released/packages/coq-tree-calculus/coq-tree-calculus.1.0.0/opam:24]: I've added a comment after license: "MIT" #

**Comment by barry-jay-personal** [comment]
Does "close pull request' mean the pull request has been accepted or rejected? I'll check in a minute

**Comment by barry-jay-personal** [comment]
I can see opam-coq-archive/released/packages/coq-tree-calculus/coq-tree-calculus.1.0.0/opam but not the other files. Have I done something wrong? What next?

**Comment by palmskog** [comment]
@barry-jay-personal the `opam` file is all that should be added to the archive. 

To install your package, all users need to know is to add the Coq repository:
```
opam repo add coq-released https://coq.inria.fr/opam/released
```
and then do:
```
opam install coq-tree-calculus
```
If they already have the Coq opam repository added to opam,they should run:
```
opam update
```

**Comment by barry-jay-personal** [comment]
Okay, thanks palmskog. I have done this easily but could not find anywhere online instructions on how to *use* a package. Finally, I guessed to try "Require Import Reflective.Tree_Calculus". Success! But how would other users figure this out. Do they need to read the opam file for the logpath and got to my github repository to find the file name? Or is there a simpler way? My default plan is to tweet as follows. 

The coq proofs for "Reflective Programs in Tree Calculus" are now available at https://github.com/barry-jay-personal/tree-calculus/ and as the opam package coq-tree-calculus (logpath Reflective). 

Thanks again for all your help, guys. It is much appreciated.

**Comment by palmskog** [comment]
@barry-jay-personal unfortunately, the documentation of how to use a package is always separate from the opam package definition. There are several options, like using the `doc` field in a package definition to point to some HTML tutorial, and using the GitHub `README.md` to provide both installation instructions and a small example. For an example, see here: https://github.com/coq-community/aac-tactics#building-and-installation-instructions

**Comment by barry-jay-personal** [comment]
Thanks for all your help!
Barry

**Pull request merged**